3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,92 @@
+
+//A future age calculator
+let yearOfBirth = 1993 ;
+
+let yearFuture = 2030 ;
+
+const age = yearFuture - yearOfBirth ;
+
+const theSentence = "You will be" + " " + age + " " +  "years old in" + " " + yearFuture+ "." ;
+
+console.log(theSentence);
+
+//A dog age calculator
+
+let dogYearOfBirth = 2020 ;
+
+let dogYearFuture = 2030 ;
+
+let shouldShowResultInDogYears = true ;
+
+const dogAgeInHumanYear = dogYearFuture - dogYearOfBirth ;
+
+const dogAgeInDogYear = dogAgeInHumanYear * 7 ;
+
+let dogAge ;
+if(shouldShowResultInDogYears) {
+    dogAge = "Your dog will be" + " " + dogAgeInDogYear + " " + "dog years old in" +  " " + dogYearFuture + "." ;
+}
+else {
+    dogAge = "Your dog will be" + " " + dogAgeInHumanYear + " " + "human years old in" +  " " + dogYearFuture + "." ;
```

Comment:
Here you could consider using template literals `` and `${}` to directly insert variables into the string. Doing this 
will make your code a bit more clean and easier to read.